### ISSUE TITLE
Question: Debug.Print vs Print vs ?

### ISSUE THREAD

**Issue by Greedquest** [WaynePhillipsEA/twinbasic#166]
The following code
``` vba
Module m
	Sub t()
		?"a"
		Print "b"
		'Debug.? "c" 'does not work, and gets a red squiggle under ?
		Debug.Print "d"
	End Sub
End Module
```
Will print:
```none
a
b
d
```
... to the console. Is there any difference between these or are they all aliases? In VB* standalone Print requires an output stream.

---

Tested under 

> Name: twinBASIC
> Id: twinbasic.twinbasic
> Description: 
> Version: 0.10.509
> Publisher: twinBASIC
> VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=twinbasic.twinbasic

**Comment by WaynePhillipsEA** [comment]
`?"a"
Print "b"`

You're right... VB6/A disallows those, except in the immediate window.  I think tB should probably do the same, as I'd prefer to see Debug.Print than either of those two shortened forms in actual code.

And I'll get the Debug.? form fixed.  Thanks!

**Comment by cristianbuse** [comment]
Still executes although warning is displayed:  
```VBA
Module m
	Sub t()
		?"a"
		Print "b"
		Debug.? "c" 'Compile warning but still executes and gets skipped
		Debug.Print "d"
	End Sub
End Module
```

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.580, thanks for reporting
